distribution/engine/THIRD-PARTY/NOTICE

```diff
@@ -36,11 +36,6 @@ The license file can be found at `licenses/APACHE2.0`.
 Copyright notices related to this dependency can be found in the directory `com.fasterxml.jackson.core.jackson-annotations-2.11.1`.
 
 
-'jline', licensed under the The BSD License, is distributed with the engine.
-The license information can be found along with the copyright notices.
-Copyright notices related to this dependency can be found in the directory `org.jline.jline-3.15.0`.
-
```

Comment:
It is weird for me that after the version bump in `build.sbt` for `jline` this notice has been removed completely instead of being updated with the updated version string etc.

The dependency is still there in `engine-runner` so it seems logical that it should be listed *somehow*.